cli/core/pkg/command/plugin_manager.go

```diff
@@ -482,7 +483,7 @@ func getRepositories() *cli.MultiRepo {
 
 // getInstalledElseAvailablePluginVersion return installed plugin version if plugin is installed
 // if not installed it returns available recommanded plugin version
-func getInstalledElseAvailablePluginVersion(p plugin.Discovered) string {
+func getInstalledElseAvailablePluginVersion(p plugin.Discovered) string { // nolint:gocritic
```

Comment:
Can this linter error be easily solved by passing argument as a pointer instead of entire object? If yes, this might be really quick thing to fix if we can fix it quickly.

Reply:
I've filed a story to fix a bunch of these linter errors en-masse - https://github.com/vmware-tanzu/tanzu-framework/issues/3470